hw/application_fpga/Makefile

```diff
@@ -451,18 +468,20 @@ view: tb_application_fpga_vcd
 #-------------------------------------------------------------------
 clean: clean_fw clean_tb
 	rm -f bram_fw.hex
-	rm -f synth.{v,json,txt} route.v application_fpga.{asc,bin,vcd} application_fpga_testfw.bin
-	rm -f tb_application_fpga.vvp synth_tb.vvp route_tb.vvp
+	rm -f synth.{v,json,txt} application_fpga.{asc,bin} application_fpga_testfw.bin
+	rm -f tb_application_fpga_sim.fst
+	rm -f tb_application_fpga_sim.fst.hier
```

Comment:
I'm missing a few of these in the `.gitignore` file
- `tb_application_fpga_sim.fst`
- `tb_application_fpga_sim.fst.hier` <- never seen this generated however. When is it generated?
- `tb_verilated/`